Makefile

```diff
@@ -287,3 +291,7 @@ toolbox: ## Create a toolbox instance with the proper Golang and Operator SDK ve
 	$(IMAGE_BUILDER) stop opendatahub-toolbox ||:
 	toolbox rm opendatahub-toolbox ||:
 	toolbox create opendatahub-toolbox --image localhost/opendatahub-toolbox:latest
+
+.PHONY: e2e-test
+e2e-test: ## Run e2e tests for the controller
```

Comment:
should this one has dependency of "image deploy" ?

Reply:
Adding that dependency will then remove the separation between deployment of operator and tests.

That means if someone deployed the operator through olm, they again will run `deploy` as part of `make e2e-test`.